playbooks/allocate_pubcloud.yml

```diff
@@ -178,7 +178,7 @@
 
     - name: Install python onto the host if it is not already present
       when:
-        - "_python_check.stdout | bool"
+        - "_python_check.stdout | trim | bool"
```

Comment:
It's always safer to ensure precedence, just in case: ``(_python_check.stdout | trim) | bool``